src/extensions/Wyam.Html/Wyam.Html.csproj

```diff
@@ -1,17 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <Description>Wyam is a simple to use, highly modular, and extremely configurable static content generator. This library provides support for parsing and manipulating HTML content.</Description>
-    <PackageTags>Wyam Static StaticContent StaticSite Blog BlogEngine HTML</PackageTags>
-    <RepositoryUrl>https://github.com/Wyamio/Wyam.git</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-  </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\..\core\Wyam.Common\Wyam.Common.csproj">
-      <PrivateAssets>all</PrivateAssets>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="AngleSharp" Version="0.9.10" />
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>Wyam is a simple to use, highly modular, and extremely configurable static content generator. This library provides support for parsing and manipulating HTML content.</Description>
+    <PackageTags>Wyam Static StaticContent StaticSite Blog BlogEngine HTML</PackageTags>
+    <RepositoryUrl>https://github.com/Wyamio/Wyam.git</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\core\Wyam.Common\Wyam.Common.csproj">
+      <PrivateAssets>all</PrivateAssets>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="AngleSharp" Version="0.9.10" />
+    <PackageReference Include="System.Net.Requests" Version="4.3.0" />
```

Comment:
Added this so I can get some SourceLink magic when I was trying to diagnose the bug. With this package it now allows for debugging inside the requests.

Reply:
Great idea. The NuGet page for the `System.Net.Requests` says
> Developers should prefer the classes in the System.Net.Http package

Would explicitly adding the `System.Net.Http` package accomplish the same SourceLink magic?

Reply:
No, System.Net.Http does not contain the System.WebRequest's types, only HttpClient.

Reply:
I was tempted to upgrade to HttpClient but decided against it.